specification/registry/xr.xml

```diff
@@ -1889,14 +1889,14 @@ maintained in the default branch of the Khronos OpenXR GitHub project.
         <enum value="2" name="XR_REMOTING_CONNECTION_STATE_CONNECTED_MSFT"/>
     </enums>
     <enums name="XrRemotingVideoCodecMSFT" type="enum">
-        <enum value="0" name="XR_REMOTING_VIDEO_CODEC_ANY"/>
-        <enum value="1" name="XR_REMOTING_VIDEO_CODEC_H264"/>
-        <enum value="2" name="XR_REMOTING_VIDEO_CODEC_H265"/>
+        <enum value="0" name="XR_REMOTING_VIDEO_CODEC_ANY_MSFT"/>
+        <enum value="1" name="XR_REMOTING_VIDEO_CODEC_H264_MSFT"/>
+        <enum value="2" name="XR_REMOTING_VIDEO_CODEC_H265_MSFT"/>
     </enums>
     <enums name="XrRemotingCertificateNameMismatchMSFT" type="enum">
-        <enum value="0" name="XR_REMOTING_CERTIFICATE_NAME_NOT_CHECKED"/>
-        <enum value="1" name="XR_REMOTING_CERTIFICATE_NAME_MATCH"/>
-        <enum value="2" name="XR_REMOTING_CERTIFICATE_NAME_MISMATCH"/>
+        <enum value="0" name="XR_REMOTING_CERTIFICATE_NAME_MISMATCH_NOT_CHECKED_MSFT"/>
+        <enum value="1" name="XR_REMOTING_CERTIFICATE_NAME_MISMATCH_MATCH_MSFT"/>
+        <enum value="2" name="XR_REMOTING_CERTIFICATE_NAME_MISMATCH_MISMATCH_MSFT"/>
```

Comment:
To prevent the awkward MISMATCH_MISMATCH, I suggest we go with XrRemotingCertificateNameValidationResultMSFT instead.